platform/darwin/src/MGLRasterStyleLayer.h

```diff
@@ -6,6 +6,27 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ The resampling/interpolation method to use for overscaling, also known as
+ texture magnification filter
```

Comment:
Nit: This would be fixed upstream in the style specification, but there’s a missing period at the end of this documentation comment and the ones for the individual values below. Not critical though.